src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/ToModelTransformers.kt

```diff
@@ -138,7 +138,7 @@ fun transform(
     isVariation = licence.variationOfId != null,
     variationOf = licence.variationOfId,
     createdByFullName = "${licence.createdBy?.firstName} ${licence.createdBy?.lastName}",
-    isInPssPeriod = licence.isInPssPeriod(),
+    isInPssPeriod = if (licence.typeCode === LicenceType.PSS) true else licence.isInPssPeriod(),
```

Comment:
Return true if the licence type is PSS.

Reply:
could this go inside the logic on the licence? (generally worth pushing the logic down as far as it can go)